examples/gallery/maps/country_polygons.py

```diff
@@ -0,0 +1,98 @@
+"""
+Highlight country, continent and state polygons
+-----------------------------------------------
+The :meth:`pygmt.Figure.coast` method can highlight country polygons
+via the ``dcw`` parameter. It accepts the country code or full
+country name and can draw its borders and add a color to its landmass.
+It's also possible to define multiple countries at once by separating
+the indiviudal names with commas.
+"""
+
+# sphinx_gallery_thumbnail_number = 1
+
+
+import pygmt
+
+fig = pygmt.Figure()
+
+fig.coast(
```

Comment:
I would adjust the appearance of the gridlines. Currently, they are a bit dominant. (Maybe this also makes sense for the second example.)

```suggestion
fig = pygmt.Figure()

# Adjust the appearance of the gridlines
pygmt.config(MAP_GRID_PEN_PRIMARY="0.1p,gray60")

fig.coast(
```